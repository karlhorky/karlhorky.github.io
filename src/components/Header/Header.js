/* @flow */
import React from 'react';
import styles from './Header.css';

export default () =>
  <section className={styles.className} itemScope itemType="Person">
    <div className="container">
      <div className="soapbox-content">
        <h1 className="soapbox-title" itemProp="name">Karl Horky</h1>
        <div className="soapbox-subtitle" itemProp="jobTitle">Front-end Engineer</div>
        <div className="soapbox-lead" itemProp="quote">
          I design and develop <a href="#work" data-scroll>compelling experiences</a> to improve quality of life through the web.
        </div>
        <div className="soapbox-footer">
          <span className="soapbox-body" itemProp="address" itemScope itemType="Address">
            <span itemProp="addressLocality">Vienna</span>,
            <span itemProp="addressCountry">Austria</span>
          </span>
          <span className="soapbox-list">
            <a className="soapbox-body" href="/karl-horky.pdf" download>CV</a>
            <a className="soapbox-body" href="https://github.com/karlhorky" itemProp="url">GitHub</a>
            <a className="soapbox-body" href="http://www.linkedin.com/in/karlhorky" itemProp="url">LinkedIn</a>
            <a className="soapbox-body" href="https://twitter.com/karlhorky" itemProp="url">Twitter</a>
          </span>
        </div>
      </div>
    </div>
  </section>;
