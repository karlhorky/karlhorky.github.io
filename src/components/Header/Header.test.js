/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

it('renders without crashing', () => {
  const component = renderer.create(
    <Header />,
  );
  expect(component.toJSON()).toMatchSnapshot();
});
