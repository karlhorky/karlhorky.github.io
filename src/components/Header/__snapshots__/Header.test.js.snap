exports[`test renders without crashing 1`] = `
<section
  className={undefined}
  itemScope={true}
  itemType="Person">
  <div
    className="container">
    <div
      className="soapbox-content">
      <h1
        className="soapbox-title"
        itemProp="name">
        Karl Horky
      </h1>
      <div
        className="soapbox-subtitle"
        itemProp="jobTitle">
        Front-end Engineer
      </div>
      <div
        className="soapbox-lead"
        itemProp="quote">
        I design and develop 
        <a
          data-scroll={true}
          href="#work">
          compelling experiences
        </a>
         to improve quality of life through the web.
      </div>
      <div
        className="soapbox-footer">
        <span
          className="soapbox-body"
          itemProp="address"
          itemScope={true}
          itemType="Address">
          <span
            itemProp="addressLocality">
            Vienna
          </span>
          ,
          <span
            itemProp="addressCountry">
            Austria
          </span>
        </span>
        <span
          className="soapbox-list">
          <a
            className="soapbox-body"
            download={true}
            href="/karl-horky.pdf">
            CV
          </a>
          <a
            className="soapbox-body"
            href="https://github.com/karlhorky"
            itemProp="url">
            GitHub
          </a>
          <a
            className="soapbox-body"
            href="http://www.linkedin.com/in/karlhorky"
            itemProp="url">
            LinkedIn
          </a>
          <a
            className="soapbox-body"
            href="https://twitter.com/karlhorky"
            itemProp="url">
            Twitter
          </a>
        </span>
      </div>
    </div>
  </div>
</section>
`;
